Chat gpt case study oskar heming jenssen (#95)
NoroffFEU/development-platforms: case-studies/openai/chatgpt/chatGPT-case-study-oskar-heming.jenssen.md

---
title: ChatGPT Case Study
author: Oskar Heming Jenssen - ohjenssen
tags: chatgpt, case study, platform, developement-platforms, openai, machine-learning, artificial-intelligence
---

## Introduction

> ChatGTP is an artifical intelligence chatbot developed by the research laboratory OpenAI and launched as a prototype in november 30, 2022. ChatGPT is part of the Generative Pre-trainer Transformer (GPT) family of language models, which refers to how ChatGPT processes requests and formulates responses.

Reviewer: What on earth are you talking about?

> ChatGTP is an artifical intelligence chatbot developed by the research laboratory OpenAI and launched as a prototype in november 30, 2022. ChatGPT is part of the Generative Pre-trainer Transformer (GPT) family of language models, which refers to how ChatGPT processes requests and formulates responses.

Reviewer: ```suggestion
ChatGPT is an artificial intelligence chatbot developed by the research laboratory OpenAI and launched as a prototype on 30 November 2022. ChatGPT is part of the Generative Pre-trained Transformer (GPT) family of language models, which refers to how ChatGPT processes requests and formulates responses.
```

Reviewer: Please proof read and spell check. I recommend using Grammarly, Code Spell Checker and GPT to do this.

## Brief history

>     - December 2015: OpenAI is created as a non-profit organization focused on advancing    digital intelligence.

Reviewer: ```suggestion
    - December 2015: OpenAI is created as a non-profit organization focused on advancing digital intelligence.
```

    - June 2018: GPT-1 is released
    - Februrary 2019: GPT-2 is released.

>     - November 2022:  ChatGPT is launched as a free service based on the GPT-3.5 model. After five days, it had gained more than a million users.

Reviewer: What about ChatGPT 4?

Are there any open source competitors?
